doc/release/0.19.0-notes.rst

```diff
@@ -106,6 +126,19 @@ substantially faster when operating on some n-dimensional arrays when n > 1.
 The largest reduction in computation time is realized in cases where the size
 of the array is small (<1k samples or so) along the axis to be filtered.
 
+`scipy.fftpack` improvements
+----------------------------
+
+Fast Fourier transform routines now accept `np.float16` inputs and upcast
+them to `np.float32`. Previously, they would raise an error.
+
+`scipy.cluster` improvements
+----------------------------
+
+Methods ``"centroid"`` and ``"median"`` of `scipy.cluster.hierarchy.linkage`
+have been significantly sped up.
```

Comment:
Maybe another sentence to add: 

    Long-standing issues with using ``linkage`` on large input data (over 16 GB) have been resolved.